docs/integrations/azure/locations.md

```diff
@@ -28,3 +28,5 @@ The configuration is a structure with two elements:
 
 - `host`: The DevOps host; only `dev.azure.com` is supported.
 - `token` (optional): A personal access token as expected by Azure DevOps.
+
+> Note: The token should just be provided as the raw token generated by Azure DevOps using the format `:{raw_token}`
```

Comment:
So, as per 

https://github.com/microsoft/typed-rest-client/blob/df510b7f690a6b59aed5d68c7ae9e15bf5c79724/lib/handlers/personalaccesstoken.ts#L23-L25 

it should just be `${raw_token}` - will update

Reply:
If I'm honest, I would like the docs to reference both that typed-rest-client and the library that implements it, so that people reading this get an understanding of how the information trickles down to the libraries that eventually use it. That should be rather transparent - more information would need to be added around the actual implementation in the plugins folder to make sure that if people update those dependencies, they also update them here - unless you want to actually specify a contract - in which case, this should also be documented.

Currently it feels a bit like, this is the way things are done based off of the dependencies - when in actual fact, we should be stating here, this is how it is done, and the translation for appropriate dependencies is done in a facade fashion.

Reply:
Sorry for the slow follow up, thanks for making the change. What ended up getting merged in will for sure help! 🚀